src/app/pages/installation/_mac_designer.twig

```diff
@@ -10,6 +10,11 @@ Use the Axon Ivy Designer <a href="/download/sprint">9.2</a> or newer to work wi
 
 {% else %}
 <h3>Mac Designer for BigSur and older</h3>
+<p style="background-color:orange; padding:20px;">
```

Comment:
orange? :D 
how does this look like?

Reply:
![grafik](https://github.com/axonivy/dev.axonivy.com/assets/5229480/b92e2e2b-254c-4a79-a1c5-1807c6aeafc7)

Reply:
Same is used if you download version 9.2 or smaller